gluster_bridge.spec

```diff
@@ -0,0 +1,75 @@
+%define pkg_name tendrl-gluster-bridge
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Common Module for All Bridges
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: GPLv2+
+Url: https://github.com/Tendrl/gluster_bridge
+
+#Requires: python-etcd
+Requires: python-dateutil >= 2.2
+Requires: python-gevent >= 1.0
+Requires: python-greenlet >= 0.3.2
+Requires: pytz
+
+%description
+Python module for Tendrl gluster bridge to manage gluster tasks.
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+
+# Remove bundled egg-info
+rm -rf %{pkg_name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# generate html docs
+%if 0%{?rhel}==6
+sphinx-1.0-build doc/source html
+%else
+sphinx-build doc/source html
+%endif
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -Dm 0644 tendrl-glusterd.service $RPM_BUILD_ROOT/usr/lib/systemd/system/tendrl-glusterd.service
+
+%post
```

Comment:
I remember Rohan commenting somewhere not to copy the sample. Rather create one with proper content. So this comment is null and void :)